src/components/cycle-section/cycle-section/LearningUnitsSection.js

```diff
@@ -0,0 +1,39 @@
+import { TabView, TabPanel } from 'primereact/tabview';
+import LearningUnitsListTab from '@components/cycle-section/learning-units-section/learning-units-section/LearningUnitsListTab';
+import LearningUnitsGraph from '@components/cycle-section/learning-units-section/learning-units-graph/LearningUnitsGraph';
+
+const LearningUnitsSection = ({
+  learningUnits,
+  successions,
+  handleLearningUnitClick,
+}) => {
+  return (
+    <TabView>
+      <TabPanel header="Mapa de pre requisitos">
+        <div style={{ display: 'flex', flexWrap: 'wrap' }}>
+          <div
+            style={{
+              height: 600,
+              width: '89.6vw',
+              margin: 15,
+              position: 'relative',
+            }}
```

Comment:
I believe this is considered an in-line style, which I believe we are not using, but I leave it up to you.